topics/fruit.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Fruit</title>
+</head>
+<body>
+    <h1>Favorite Fruit</h1>
+   
+    <ul>
+		<li> Coconut</li>
```

Comment:
Let's fix this whitespace, I would recommend opening the file here in Github and fixing that issue!!